schemas/test/en/test_numbers.json

```diff
@@ -290,6 +290,67 @@
                                 "title": "Please enter test values (none mandatory)",
                                 "type": "General"
                             }
+                        },
+                        {
+                            "type": "Question",
```

Comment:
Although this doesn't follow the pattern of the rest of the schema, I wasn't happy that I could exercise detail_answer correctly. The previous block already had a dependency on set-maximum, it was feasible that was the reason for block being removed and therefore details_answer wasn't doing it. Keeping it completely separate proved the code was working correctly for detail-answer